src/skia/ColorFilter.cpp

```diff
@@ -130,14 +135,21 @@ py::class_<SkColorMatrix>(m, "ColorMatrix")
     ;
 
 py::class_<SkColorFilters>(m, "ColorFilters")
+/*
```

Comment:
Revisit, right?

Reply:
This particular one, has a revisit at tests/test_colorfilter.py at line 50. So it will be looked at, at some point (no promises about time scale...), when the test get looked at. In the m116-public-try branch on my repo, I have a few, which either get re-enbled (if the code can be modified/updated to work), or switch to xfail, and a n additional comment added above the "/*" saying or quoting upstream release notes.

In general, it is possible I have commented out code that don't compile anymore without an associated "revisit" test. At some point I'll diff against v87.5 and double-check. At least my comment-out style is consistent. So they can be grep with `grep '/\*`. I checked it that way in the "no workaround" lineage, with "git diff v87.5... | grep '/\*'" until it got too inconvenient to insist on a no-workaround lineage...

Reply:
So in the end, I expect every one of "grep -1 "/\*" src/skia/*" to show a previous line about why it was gone. Don't know how long it will take though. :-(. I hope this is an answer you can accept.